src/signUp.js

```diff
@@ -0,0 +1,47 @@
+import { TEST_AUTH, ERROR_MESSAGES } from "./constants/VALIDATION.js";
+import {
+  emailError,
+  emailInput,
+  pwError,
+  pwInput,
+  pwConfirmError,
+  pwConfirmInput,
+  form,
+  pwToggle,
+  pwConfirmToggle,
+  validateEmail,
+  validatePw,
+  validatePwConfirm,
+  eyeToggle,
+  showError,
+} from "./auth.js";
+
+const isDuplicateEmail = () => {
+  if (emailInput.value.trim() === TEST_AUTH.email) {
+    showError(emailError, emailInput, ERROR_MESSAGES.email_duplicate);
+    return true;
+  } else return false;
+};
+
+const handleSignUp = (e) => {
+  e.preventDefault();
+  if (
+    !isDuplicateEmail() &&
+    validateEmail() &&
+    validatePw() &&
+    validatePwConfirm()
+  ) {
+    alert("회원가입 성공!");
+    window.location.href = "folder.html";
+  }
+};
```

Comment:
위 리뷰를 참고하면 이런 식으로 작성할 수 있겠네요.
```suggestion
const handleSignUp = (e) => {
  e.preventDefault();
  
  const isAllValid = !isEmailDuplicate && isEmailValid && isPwValid && isPwConfirmValid;
  
  if (isAllValid) {
    alert("회원가입 성공!");
    window.location.href = "folder.html";
  }
};
```